vmcnet/models/core.py

```diff
@@ -12,11 +12,28 @@
 from vmcnet.utils.slog_helpers import slog_sum
 from vmcnet.utils.typing import Array, ArrayList, PyTree, SLArray, ParticleSplit
 from .weights import WeightInitializer, get_bias_initializer, get_kernel_initializer
+from flax.linen import SelfAttention  # noqa
 
 Activation = Callable[[Array], Array]
 SLActivation = Callable[[SLArray], SLArray]
 
 
+def _transformer_mix(
```

Comment:
Now that I think of it I don't think this should really be in the core.py file, since I don't foresee it being a core utility used in many different places. Could we put it in the equivariance.py file either as a stand-alone function or as a method of `FermiNetOneElectronLayer`?